### ISSUE TITLE
Web: Make NewsBlur WCAG compliant

### ISSUE THREAD

**Issue by lionirdeadman** [samuelclay/NewsBlur#1524]
There is many issues with regards to WCAG compliance for accessibility. These can all be seen from Firefox's Accessibility tab.

- [ ] Lack Keyboard shortcuts attributes
- [ ] Lack Text alternative for UI elements
- [ ] Low contrast

**Comment by samuelclay** [comment]
I'd love to see some specifics on what has left to be done. I gathered from the WCAG docs that having keys be remappable and entirely disable-able is necessary, so I've retitled this ticket to do so.

As for the text alternative for UI elements, all elements should have a label. Is there a specific attribute we're missing?
